### ISSUE TITLE
How to install deforum on comfyui

### ISSUE THREAD

**Issue by ca3gamedev** [comfyanonymous/ComfyUI#1134]
Hi, just want to know how to make animations in comfyui.

**Comment by jaideep2** [comment]
no way of doing this right now

**Comment by sanguivore-easyco** [comment]
https://github.com/alt-key-project/comfyui-dream-project - just found this in the same search that brought me to this issue, haven't played with it myself yet

**Comment by jaideep2** [comment]
Hey @sanguivore-easyco looks promising, but I see the image gets darker and darker because of the design choices. Here is another one that uses a different approach https://github.com/Kosinkadink/ComfyUI-AnimateDiff-Evolved

**Comment by sanguivore-easyco** [comment]
> Hey @sanguivore-easyco looks promising, but I see the image gets darker and darker because of the design choices. Here is another one that uses a different approach https://github.com/Kosinkadink/ComfyUI-AnimateDiff-Evolved

My main issue with this one is that it doesn't seem able to support SDXL and exports its results as gifs with lots of visual artifacts (that and I do really like the ability to music-sync things like in deforum, which I don't think AnimateDiff can support)

**Comment by jaideep2** [comment]
Agreed! Let me know if you happen to come across anything with SDXL. I am currently using https://github.com/neggles/animatediff-cli because it works with my GPU that has 12GB VRAM.

**Comment by sanguivore-easyco** [comment]
> > > Hey @sanguivore-easyco looks promising, but I see the image gets darker and darker because of the design choices. Here is another one that uses a different approach https://github.com/Kosinkadink/ComfyUI-AnimateDiff-Evolved
> > 
> > 
> > My main issue with this one is that it doesn't seem able to support SDXL and exports its results as gifs with lots of visual artifacts (that and I do really like the ability to music-sync things like in deforum, which I don't think AnimateDiff can support)
> 
> You can change the output for mp4 or some other options. But agree no sdxl is a shame

There's a more recent model for SDXL support in the huggingface repo for AnimateDiff now: https://huggingface.co/guoyww/animatediff/tree/main cc @jaideep2 

And also with the help of https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite and ControlNets, you could possibly sync with music like Deforum in a sense.

**Comment by XmYx** [comment]
Check our new Deforum Node package here:
https://github.com/XmYx/deforum-comfy-nodes
